recipes-hosted-login/mender-hosted-login/mender-hostedlogin-0.1/menderLogin.py

```diff
@@ -189,6 +190,15 @@ def _logIn_clicked(self):
                 f_tmp = open("/tmp/mender.conf.tmp", "w+")
                 # Rewrite mender.conf
                 json.dump(data, f_tmp, indent=2, sort_keys=True)
+
+                # Also write the tenant token to a persistent file
+                # this will be restored into mender.conf by the mender.service
+                # launch script.
+                p_tmp = open("/data/mender/hosted-mender-settings.dat", "w")
```

Comment:
Will this file be writable by non-root?

Reply:
Well, /data and /data/mender have group write permissions and it all seems to work.  This file is only ever created by this script so I'm not too worried about it being created by someone else.